task5.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Registration Form</title>
+  </head>
+  <body>
+    <h1>Registration Form</h1>
+    <form>
+      <label for="name">Name:</label>
+      <input type="text" id="name" name="name" required><br><br>
+
+      <label for="dob">Date of Birth:</label>
+      <input type="date" id="dob" name="dob" required><br><br>
+
+      <label for="gender">Gender:</label>
+      <input type="radio" id="male" name="gender" value="male" required>
+      <label for="male">Male</label>
+      <input type="radio" id="female" name="gender" value="female">
+      <label for="female">Female</label>
+      <input type="radio" id="other" name="gender" value="other">
+      <label for="other">Other</label><br><br>
+
+      <label for="tech">Tech Stacks:</label>
+      <input type="checkbox" id="ai" name="tech" value="ai">
+      <label for="ai">AI</label>
+      <input type="checkbox" id="ml" name="tech" value="ml">
+      <label for="ml">ML</label>
+      <input type="checkbox" id="cloud" name="tech" value="cloud">
+      <label for="cloud">Cloud</label>
+      <input type="checkbox" id="web" name="tech" value="web">
+      <label for="web">Web</label>
+      <input type="checkbox" id="mobile" name="tech" value="mobile">
+      <label for="mobile">Mobile</label>
+      <input type="checkbox" id="devops" name="tech" value="devops">
+      <label for="devops">DevOps</label>
+      <input type="checkbox" id="system" name="tech" value="system">
+      <label for="system">System</label><br><br>
+
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" required><br><br>
+
+      <label for="phone">Phone No:</label>
+      <input type="tel" id="phone" name="phone" required><br><br>
+
+      <label for="security">Security Question:</label>
+      <select id="security" name="security">
+        <option value="color">What is your favorite color?</option>
+        <option value="food">What is your favorite food?</option>
+        <option value="movie">What is your favorite movie?</option>
+        <option value="book">What is your favorite book?</option>
+      </select><br><br>
+
```

Comment:
There are some inputs missing
- [ ] Input for Security Answer
- [ ] Resume file upload
- [ ] Reset button